src/Core/Models/SystemRegisters/RegisteredSystemDTO.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Altinn.Platform.Authentication.Core.Models;
+
+[ExcludeFromCodeCoverage]
+public record class RegisteredSystemDTO
+{
+    /// <summary>
+    /// A unique External Id for this System, in human-readable string format.    
+    /// </summary>
+    [Required]
+    public string SystemId { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Organization number of the system Vendor that offers the product (system)
+    /// </summary>
+    [Required]
+    public string SystemVendorOrgNumber { get; set; }
+
+    /// <summary>
+    /// Organization number of the system Vendor that offers the product (system)
+    /// </summary>
+    [Required]
```

Comment:
Can we stick to the required keyword instead of the attribute [Required]? i see we use both here